.github/actions/build/action.yml

```diff
@@ -0,0 +1,120 @@
+name: build
+description: 'build connaisseur image'
```

Comment:
Do we do Capitalization in actions?

Reply:
bc below it's `Install Cosign`

Reply:
don't think we have much consistency. I am planning to do a full refactoring after this is merged. Also the above is a description and `install cosign` is a job name....